docs: add travel funding procedure (#2121)
asyncapi/community: docs/020-governance-and-policies/travel-guide.md

---
title: Community Travel Funding
weight: 27
---

This document explains the rules, priorities, and costs we cover using the conference travel budget.

### How We Choose Who Travels

We use the following priority list to decide who gets travel money. This list is for people who already have a visa or have a clear plan to get one in time.

1.  People Hired by AsyncAPI (Contracted Staff): These people are highest on the list. They also get a larger budget that should cover their travel, visa, and other costs. There is no set limit, but they must check and agree on the budget needed with the available total funds.
2.  People Who Have Not Received Funding Before: If you have never received travel funding from us but are on priority list, you get priority over people who have received it in the past.
3.  People with Key Event Jobs: People who agree to help with important duties at the event:
    * Master of Ceremony (MC): Up to two people are needed for this job.
    * API Standards Booth Lead: Up to two people are needed to manage the [API Standards booth](https://groups.io/g/api-standards).
4.  Technical Steering Committee (TSC) Members
5.  Ambassadors and Maintainers

> 6.  Speakers

Reviewer: ```suggestion
1. People Hired by AsyncAPI (Contracted Staff): These people are the highest priority. They receive a larger budget covering travel, visa, and other costs. The budget is flexible, but they must agree on the required amount with the available funds.
2. People Who Have Not Received Funding Before: The people who never received travel funding are higher in priority over people who have received it in the past.
3. People with Key Event Jobs: People who agree to help with important duties at the event:
    - Master of Ceremony (MC): Up to two people are needed for this job.
    - API Standards Booth Lead: Up to two people are needed to manage the [API Standards booth](https://groups.io/g/api-standards).
4. Technical Steering Committee (TSC) Members
5. Ambassadors and Maintainers
6. Speakers
```

> **A Note on Special Cases:** We can change these rules for special situations. For example, for the 2025 conference in India, we chose to only give money to community members who live in India. This way, we could support many more people. The money for one long international trip paid for five local trips. This allowed to us to connect large group of community members that for years have never seen each other in person.

### Problems with Visas and Planning

Getting a visa can take a long time and be stressful, especially when decisions come very late. This makes planning difficult for everyone. To help with smooth planning:

* Tell Us Early: If you want to go to an event, need a visa you don't have, and want our funding, you must tell us 6 months before the event date.
* Start Right Away: Once we confirm that we have money for your travel, you must start the visa application process **immediately**.
* We Will Help: We will help you get the necessary papers, like official invitation letters. We want to support everyone fairly while making sure our events are organized well.

### How Much We Can Pay

> The most money we can pay for one person's travel costs is based on where the travel takes place:

Reviewer: ```suggestion
The maximum we can pay for one person's travel depends on the destination.
```
